### PR TITLE
Make a Cairo 1 program for testing SquareRoot & U256SquareRoot hint 

### LINKED ISSUE #1121: Make a Cairo 1 program for testing SquareRoot hint

### PR DESCRIPTION
**Pull request by rcatalan98**
# Make a Cairo 1 program for testing SquareRoot hint 
## Adds Cairo 1 programs to test SquareRoot hint
The PR adds multiple Cairo 1 programs to test SquareRoot and U256SquareRoot hints  This includes test for u8, u16, u32, u64, u128 and u256.
## Checklist
- [x] Linked to Github Issue
- [x] Unit tests added
- [ ] Integration tests added.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.
  - [ ] CHANGELOG has been updated.

Closes #1121 and #1122

### CODE REVIEW

File: cairo_programs/cairo-1-contracts/u128_sqrt.cairo [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+
+    #[external]
+    fn sqrt(num: felt252) -> felt252 {
+        let num_in_u128: u128 = num.try_into().unwrap();
+        let a: u64 = u128_sqrt(num_in_u128);
+        let to_return: felt252 = a.into();
+        to_return
+    }
+}
```
Comment (pefontana) [inline cairo_programs/cairo-1-contracts/u128_sqrt.cairo:16]: @rcatalan98 can you add an endline to the cairo files

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 #### Upcoming Changes
 
+* Add support for SquareRoot and U256SquareRoot under `cairo-1-hints` [#1104](https://github.com/lambdaclass/cairo-rs/pull/1104)
```
Comment (pefontana) [inline CHANGELOG.md:5]: I think it is not necessary to update the changelog as it only test.
Lets talk it tomorrow in the daily

**Review by pefontana — APPROVED** [review]
Nice tests @rcatalan98 !

File: CHANGELOG.md (hunk 2 above)
Comment (pefontana) [reply CHANGELOG.md:5]: remove it

**Review by juanbono — APPROVED** [review]

**Review by Oppen — APPROVED** [review]

**Pull request merged**